testsuite/MDAnalysisTests/coordinates/test_imd.py

```diff
@@ -0,0 +1,535 @@
+"""Test for MDAnalysis trajectory reader expectations
+"""
+
+import importlib
+import pickle
+import sys
+from types import ModuleType
+from weakref import ref
+
+import pytest
+import numpy as np
+from numpy.testing import (
+    assert_allclose,
+    assert_almost_equal,
+    assert_array_almost_equal,
+    assert_equal,
+)
+
+import MDAnalysis as mda
+from MDAnalysis.coordinates.IMD import (
+    HAS_IMDCLIENT,
+    MIN_IMDCLIENT_VERSION,
+    IMDReader,
+)
+from MDAnalysis.transformations import translate
+
+if HAS_IMDCLIENT:
+    import imdclient
+    from imdclient.tests.server import InThreadIMDServer
+    from imdclient.tests.utils import (
+        create_default_imdsinfo_v3,
+        get_free_port,
+    )
+
+from MDAnalysisTests.coordinates.base import (
+    assert_timestep_almost_equal,
+    BaseReference,
+    MultiframeReaderTest,
+)
+from MDAnalysisTests.datafiles import (
+    COORDINATES_H5MD,
+    COORDINATES_TOPOLOGY,
+    COORDINATES_TRR,
+)
+
+
+def test_IMDCLIENT_import(monkeypatch):
```

Comment:
I merged the import test into the normal tests. This leads to test failures because the test is apparently not cleaning up its environment sufficiently well.

This should be corrected because later on it may lead to even harder debug issues in the tests and we don't want to knowingly accrue this technical debt.